developer-guide.md

```diff
@@ -155,7 +165,7 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
   This means that large production sites can try out experimental features, *but* they must take
   care to limit usage to a suitable portion of their traffic.
 
-  ### 20. What are the options for usage retrictions on tokens? <a name="usage-restrictions"/>
+  ### 20. What are the options for usage retrictions on tokens? <a name="usage-restrictions"></a>
```

Comment:
<a> is not considered void element so we should explicitly write a closing tag, otherwise it's treated with a starting tag without an end, making the next bit of text a link.